setup.py

```diff
@@ -28,14 +28,16 @@
 
 setup(
     name='Flask-OpenID',
-    version='1.2.5',
+    version='1.2.6',
     url='http://github.com/mitsuhiko/flask-openid/',
     license='BSD',
-    author='Armin Ronacher, Patrick Uiterwijk',
-    author_email='armin.ronacher@active-4.com, puiterwijk@redhat.com',
+    author='Armin Ronacher, Patrick Uiterwijk, Jarek Potiuk, Jason R Coombs, Emmanuel Bavoux',
+    author_email='armin.ronacher@active-4.com, puiterwijk@redhat.com, jarek@potiuk.com, '
+                 'jaraco@jaraco.com, emmanuel.bavoux@free2move.com',
```

Comment:
I'm personally not a fan of this pattern. If you do include it, please format my name with a `.`:

```suggestion
    author='Armin Ronacher, Patrick Uiterwijk, Jarek Potiuk, Jason R. Coombs, Emmanuel Bavoux',
    author_email='armin.ronacher@active-4.com, puiterwijk@redhat.com, jarek@potiuk.com, '
                 'jaraco@jaraco.com, emmanuel.bavoux@free2move.com',
```